venv~/bin/Activate.ps1

```diff
@@ -0,0 +1,241 @@
+<#

```

Comment:
I really suggest adding `venv` to your `.gitignore` file.  No one wants to see this in a PR.